### PR TITLE
Branding and Currency Unit

### PR DESCRIPTION
**Pull request by mkg20001**
Replace the Bitcoin logo with the LBRY logo and replace BTC with LBRY

### CODE REVIEW

File: src/amount.cpp [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 #include "tinyformat.h"
 
-const std::string CURRENCY_UNIT = "BTC";
+const std::string CURRENCY_UNIT = "LBRY";
```
Comment (tpruvot) [inline src/amount.cpp]: its LBC no ?

Comment (mkg20001) [inline src/amount.cpp]: Fixed in f84c5a2d

**Comment by lyoshenka** [comment]
@kaykurokawa please review

**Comment by lyoshenka** [comment]
Merged by 800bb51fbfbff3435f4780609c7c6b86df986710

**Pull request closed without merging**